kubernetes/samples/scripts/create-weblogic-domain/model-in-image/README.md

```diff
@@ -1,56 +1,44 @@
+This sample demonstrates the WebLogic Kubernetes Operator "Model in Image" feature. Model in Image allows you to supply a Weblogic Deploy Tool (WDT) model that the operator automatically expands into a full domain home during runtime. This eliminates the need to pre-create a WebLogic domain home prior to deploying your domain resource.
 
-WDT models are a convenient and succinct alternative to WebLogic configuration scripts. They compactly define a WebLogic domain via yaml files, plus support for application archives. The WDT model 
- format is 
- described in [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling).
+WDT models are a convenient and succinct alternative to WebLogic configuration scripts and templates. They compactly define a WebLogic domain via yaml files, plus support for application archives. The WDT model format is described in the open source [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) GitHub project.
  
-When using ```Model In Image```, you can specify the WDT models and archives in the image. In addition, you can also 
-provide additional models and variable files in a ```Kubernetes Configuration Map (configmap)```.  The WDT artifacts 
-will be 
-combined together and used by the ```WebLogic Deploy Tool``` to generate the domain.  Life cycle updates can be 
-applied to the image or the configmap after initial deployment.
+When using `Model In Image`, you can include your WDT models, WDT archives, and the WDT install in your image. In addition, you can also provide additional models and variable files in a Kubernetes configuration map (configmap). When you deploy your domain resource, the operator will combine the WDT artifacts and then run the WDT tooling to generate the domain. Life cycle updates can be applied to the image or the configmap after initial deployment.
 
-This sample demonstrates deploying a WebLogic servlet application within a basic `WLS` domain, a Oracle Fusion 
-Middleware Infrastructure `JRF` domain, or a `RestrictedJRF` domain. The `JRF` domain path through the sample includes additional steps for deploying a database and initializing the database using the RCU tool. `JRF` domains are used by Oracle products that layer on top of WebLogic Server such as SOA, OSB, and FA. `RestrictedJRF` domains are used by other Oracle layered products such as CGBU.
+This sample demonstrates deploying a WebLogic servlet application within a basic `WLS` domain, an Oracle Fusion Middleware Infrastructure Java Required Files (`JRF`) domain, or a `RestrictedJRF` domain. The `JRF` domain path through the sample includes additional steps for deploying an infrastructure database and initializing the database using the Repository Creation Utility (RCU) tool. `JRF` domains are used by Oracle products that layer on top of WebLogic Server such as SOA, OSB, and FA. `RestrictedJRF` domains are used by other Oracle layered products such as CGBU.
```

Comment:
Is CGBU a division name or product name?  I believe the initial goal is simplify some products like OHS that needs part of JRF functionalities for security but don't want a database approach.  https://docs.oracle.com/middleware/12213/webtier/administer-ohs/GUID-11F66F85-C131-4F87-9989-27FF2AACED76.htm#HSADM12062

Reply:
updated